packages/block-editor/src/components/block-alignment-control/stories/block-alignment-toolbar.story.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { BlockAlignmentToolbar } from '../';
+
+/**
+ * The `BlockAlignmentToolbar` component is used to render block alignment options in the editor. The different alignment options it provides are `left`, `center`, `right`, `wide` and `full`.
+ */
+const meta = {
+	title: 'BlockEditor/BlockAlignmentToolbar',
+	component: BlockAlignmentToolbar,
+	argTypes: {
+		value: {
+			control: { type: null },
+			defaultValue: 'undefined',
```

Comment:
[`defaultValue` is deprecated](https://storybook.js.org/docs/7/api/arg-types#defaultvalue) so let's remove it. It should not affect behavior.